Gaijinizer patch/Map125.txt

```diff
@@ -284,67 +299,77 @@ Here you go!#MTLed
 完全に、あれで仕留める気だったのに……
 > CONTEXT: Map125/events/9/pages/0/198/Dialogue/hyakki
 > CONTEXT: Map125/events/9/pages/1/198/Dialogue/hyakki
-I went to ............ I wanted to finish it
-completely, though ...#MTLed
+What a mess...
+
+I really wanted to bring her down and finish
+this...#Ron
 > END STRING
 
 > BEGIN STRING
 あ、ありがとう……
 
 助けてくれて………
 > CONTEXT: Map125/events/9/pages/0/203/Dialogue/kazuya
-Oh, thank you ...... Help me .........#MTLed
+T-Thanks...
+
+For saving me...#Ron
 > END STRING
 
 > BEGIN STRING
 あんな攻撃でもピンピンしてる化物だったなんて…
 
 僕１人だったらどうなってたことか…
 > CONTEXT: Map125/events/9/pages/0/207/Dialogue/kazuya
-It's such an Attack but it's a sudden change ...
-What happened if I was alone ...#MTLed
+Being attacked by such a powerful monster...
+
+I can't imagine what would've happened if I'd
+been alone...#Ron
 > END STRING
 
 > BEGIN STRING
 言ったろ？加勢するって♪
 
 \\N[1]はアタシのお得意様なんだから！
 > CONTEXT: Map125/events/9/pages/0/212/Dialogue/hyakki
-Did you say that? I will support you♪
-\\N[1] is good at 'I'!#MTLed
+I told ya, didn't I? I've got ya back. ♪
+
+You're my valued customer after all!#Ron
 > END STRING
 
 > BEGIN STRING
 ………とにかく、奴が動けない今がチャンスだ。
 
  別の道を探して、早いとこ港へ…………
 > CONTEXT: Map125/events/9/pages/0/216/Dialogue/hyakki
-... Anyway, now is the chance that he can not
-move. Find another way and go early to the port
-......#MTLed
+...Anyway, this is our chance now while she can't
+move. Look for another route and quickly get to
+the harbor.#Ron
 > END STRING
```

Comment:
Flow (This seems more like advice to Kazuya, Hyakki has no issue getting around):
>_**...Anyways, this is your chance while she can't
move. Look for another route and quickly get to
the harbor.**_